File: Python/백준/Bronze/16099. Larger Sport Facility/Larger Sport Facility.py
q=int(input())
for i in range(q):
    a,b,c,d = map(int,input().split())
    if(a*b>c*d):
        print('TelecomParisTech')
    elif(a*b==c*d):
        print('Tie')
    else:
        print('Eurecom')

File: C#/백준/Bronze/16099. Larger Sport Facility/Larger Sport Facility.cs
using System;

namespace q{
    class w{
        static void Main(){
            int q = int.Parse(Console.ReadLine());
            for (int i =0; i<q; i++){
            long[] a = Array.ConvertAll(Console.ReadLine().Split(),long.Parse);
            if(a[0]*a[1]>a[2]*a[3]){
                Console.WriteLine("TelecomParisTech");
            }
            else if(a[0]*a[1]==a[2]*a[3]){
                Console.WriteLine("Tie");
            }
            else{
                Console.WriteLine("Eurecom");
            }
            }
        }
    }
}